package.json

```diff
@@ -27,7 +27,7 @@
     "clean:docs": "rimraf dist",
     "clean:preview": "nx clean storybook",
     "cleaner": "nx run-many --target clean --projects",
-    "compare": "nx run-many --verbose --target compare --projects",
+    "compare": "node ./tasks/compare-compiled-output.js",
```

Comment:
Is this not configurable with Nx?

Reply:
It is not - nx is focused solely on parallelization. In this, we want to run the compare in parallel but report on the results only once. That last step is not easily set-up with nx but the script already has this ability.